modules/webm/libvpx/SCsub

```diff
@@ -263,14 +263,15 @@ if env["platform"] == 'uwp':
     else:
         webm_cpu_x86 = True
 else:
+    is_ios_x86 = (env["platform"] == 'iphone' and env["ios_sim"] == "yes")
     is_android_x86 = (env["platform"] == 'android' and env["android_arch"] == 'x86')
     if is_android_x86:
         cpu_bits = '32'
     if osx_fat:
         webm_cpu_x86 = True
     else:
-        webm_cpu_x86 = (cpu_bits == '32' or cpu_bits == '64') and (env["platform"] == 'windows' or env["platform"] == 'x11' or env["platform"] == 'osx' or env["platform"] == 'haiku' or is_android_x86)
-        webm_cpu_arm = env["platform"] == 'iphone' or env["platform"] == 'bb10' or (env["platform"] == 'android' and env["android_arch"] != 'x86')
+        webm_cpu_x86 = (cpu_bits == '32' or cpu_bits == '64') and (env["platform"] == 'windows' or env["platform"] == 'x11' or env["platform"] == 'osx' or env["platform"] == 'haiku' or is_android_x86 or is_ios_x86)
+        webm_cpu_arm = (env["platform"] == 'iphone' and env["ios_sim"] != "yes") or env["platform"] == 'bb10' or (env["platform"] == 'android' and env["android_arch"] != 'x86')
```

Comment:
You could use `and !is_android_x86 ... and !is_ios_x86` here to say two accesses to the env dict. #PointlessOptimisationButHeyWhyNot :P